src/App.js

```diff
@@ -1,18 +1,28 @@
 import React from 'react';
 
+var todoList = [
+  {
+    id: 1,
+    title: "Wake up"
+  },
+  {
+    id: 2,
+    title: "Take a shower"
+  },
+  {
+    id: 3,
+    title: "Walk the dog"
+  }
+];
+
 function App() {
-  return (
-    <div style={{ textAlign: 'center' }}>
-      <header>
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a href="https://reactjs.org" target="_blank" rel="noopener noreferrer">
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
-}
+  return(
```

Comment:
There is some data missing. What kind of list is this?

Reply:
Not sure what you are talking about. But I added the header.

Reply:
That's exactly what I meant. Thank you!